lib/gslab_model/test/test.ExampleModel.R

```diff
@@ -0,0 +1,27 @@
+source("ExampleModel.R")
```

Comment:
I think that leaving a few descriptive comments and shortening some lines could improve this script's readability.